uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -0,0 +1,39 @@
+package com.aws.greengrass.steps;
+
+import io.cucumber.guice.ScenarioScoped;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+@ScenarioScoped
+public class MqttControlSteps {
+
+    @And("the control component configured:")
+    public void configureControl(String configurationSpec){
+        //@TODO Implement method
+    }
+
+    @When("Client {word} subscribe to {word} qos {int}")
+    public void subscribe(String clientId, String topic, int qos){
+        //@TODO Implement method
+    }
+
+    @When("Client {word} publish to {word} qos {int} message:")
+    public void publish(String clientId, String topic, int qos, String message){
+        //@TODO Implement method
+    }
+
+    @When("I add {string} ggad with the following policy to the CDA configuration")
```

Comment:
What that means ?

Reply:
This is a step from GGAD scenario. 
CDA means - Client Device Association
This step includes:
1. IAM policy configuration for client device to grant access to mqtt
2. Add client device to CDA

Reply:
fixed 7bec4aab4bf10f182d29bef87204a704dac3bdce

Reply:
fixed 7bec4aab4bf10f182d29bef87204a704dac3bdce